File: practice/algorithms/sorting/intro_to_tutorial_challenges/IntroToTutorialChallenges.java
import java.util.Scanner;

public class IntroToTutorialChallenges
{
	public static void main(String[] args)
	{
		Scanner stdin = new Scanner(System.in);
		int v = Integer.parseInt(stdin.nextLine());
		int size = Integer.parseInt(stdin.nextLine());
		String[] numbers = stdin.nextLine().split(" ");
		System.out.println(getPosition(numbers, v));
		
		stdin.close();
	}
	
	private static int getPosition(String[] numbers, int v)
	{
		for(int i = 0; i < numbers.length; i++)
		{
			if(v == Integer.parseInt(numbers[i]))
			{
				return i;
			}
		}
		return 0;
	}
}

File: practice/algorithms/sorting/intro_to_tutorial_challenges/IntroToTutorialChallenges.scala
import scala.io.Source

object IntroToTutorialChallenges extends App {
  val lines = Source.stdin.getLines().toList
  val searchedValue = lines(0).toInt
  val array = lines(2).split(" ").map(_.toInt)

  val valPosition = getPosition(array, searchedValue)
  println(valPosition)

  def getPosition(array: Array[Int], value: Int): Int = {
    array.indexOf(value)
  }
}